src/main/domain/status/InMemoryStatusRepository.js

```diff
@@ -0,0 +1,16 @@
+import {Status} from './Status'
+import {StatusRepository} from './StatusRepository'
+
+/**
+ * @implements {StatusRepository}
+ */
+export class InMemoryStatusRepository extends StatusRepository {
```

Comment:
maybe this file should be in `infrastructure` as an inmemory implementation instead of being in the `domain` package

Reply:
Done!